simple-nextjs-app/README.md

```diff
@@ -0,0 +1,14 @@
+Simple [Next.js](https://nextjs.org/) project that expects POST requests from [PayButton](paybutton.org) and validates their authenticity.
+
+## Getting Started
+
+1. Get you user's public key by logging in to [PayButton](paybutton.org) and acessing the **Account** view.
+2. Change `.env` to include your public key, e.g. `PUBLIC_KEY=302a300506032b6570032100615255850442064c84af43736133257cc0bf57a0da5c92baeb4be4856c43c420`
+3. Run the server with `yarn dev`.
+
+Now you should configure a PayButton Trigger by going into [PayButton](paybutton.org), creating a PayButton, clicking on it, and putting on the form "When a transaction is received...":
```

Comment:
1. It's necessary that they do it, so let's use direct language -- Here just removing "Now you should" is enough.
2. Can probably improve the wording of "putting on the form".

Reply:
1. Indeed, done.
2. Changed